src/test/deployer.t.sol

```diff
@@ -17,7 +17,13 @@ pragma solidity >=0.5.15 <0.6.0;
 
 import "ds-test/test.sol";
 import "./../deployer.sol";
-import "tinlake/borrower/fabs/pile.sol";
+
+import { TitleFab } from "./borrower/fabs/title.sol";
+import { PileFab } from "./borrower/fabs/pile.sol";
```

Comment:
why are all these imports here? also the path is incorrect, this is why the Travis build is failing.